Resources/Prototypes/SimpleStation14/Catalog/Fills/Crates/food.yml

```diff
@@ -0,0 +1,33 @@
+- type: entity
+  id: CrateFoodExoticMeats
+  name: Exotic meats crate
+  description: "Contains a variety of uncooked, exotic meats. Eat at your own peril."
+  parent: CrateFreezerSecure
+  components:
+  - type: StorageFill
+    contents:
+    - id: FoodMeatFish
+      amount: 6
+    - id: FoodMeatBear
+      amount: 4
+    - id: FoodMeatPenguin
+      amount: 4
+    - id: FoodMeatRat
+      amount: 4
+    - id: FoodMeatLizard
+      amount: 2
+    - id: FoodMeatSpider
+      amount: 2
+    - id: FoodMeatXeno
+      prob: 0.01
```

Comment:
I would make far more of these fall under a probability, so there's not quite as much meat per crate, but more variety.

Reply:
Hm is it possible to make them count towards a total amount? Or perhaps give it a one or the other kinda chance?

Reply:
Don't believe so. See maints lockers, for instance.